data-prepper-plugins/aggregate-processor/src/main/java/com/amazon/dataprepper/plugins/processor/aggregate/AggregateActionSynchronizer.java

```diff
@@ -49,7 +49,7 @@ private AggregateActionSynchronizer(final AggregateAction aggregateAction, final
         this.actionConcludeGroupEventsProcessingErrors = pluginMetrics.counter(ACTION_CONCLUDE_GROUP_EVENTS_PROCESSING_ERRORS);
     }
 
-    Optional<Event> concludeGroup(final AggregateIdentificationKeysHasher.IdentificationHash hash, final AggregateGroup aggregateGroup) {
+    Optional<Event> concludeGroup(final AggregateIdentificationKeysHasher.IdentificationHash hash, final AggregateGroup aggregateGroup, final boolean isShuttingDown) {
```

Comment:
It could be good to rename this to `forceConclude` or something similar. There is all that the flag does, so the name is more accurate, and there may be other scenarios where we want to force a conclude.

Reply:
Will change